packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -63,7 +63,23 @@ class FileSystemUtils {
   ///
   /// On Windows it replaces all '\' with '\\'. On other platforms, it returns the
   /// path unchanged.
-  String escapePath(String path) => _platform.isWindows ? path.replaceAll(r'\', r'\\') : path;
+  String escapePath(String path) {
+    if (_platform.isWindows) {
+      path = path
+          .replaceAll(r'\', r'\\')
+          .split(';')
```

Comment:
Why are we splitting on `;`? This API is only mean to handle individual paths, not the contents of the `PATH` variable.